src/Sulu/Bundle/AudienceTargetingBundle/Controller/TargetGroupController.php

```diff
@@ -274,6 +274,7 @@ private function deserializeData($data)
             $this->targetGroupRepository->getClassName(),
             'json'
         );
+        $this->entityManager->clear();
```

Comment:
I find this very strange that we are calling a setter on a serialize subscriber here:

https://github.com/sulu/sulu/blob/e3318df4ed1a13229dd86d8ff480fb3e56af3219/src/Sulu/Bundle/AudienceTargetingBundle/Serializer/Subscriber/TargetGroupSerializeSubscriber.php#L57

Instead we should add it to serialization not manipulate entities on serialization.

Reply:
Oh sorry we are in deserialize here 🙈